inst/schema/PlottingMetadataResponse.schema.json

```diff
@@ -3,7 +3,11 @@
   "definitions": {
     "choropleth_metadata": {
       "type": "array",
-      "items": { "$ref": "ChoroplethIndicatorMetadata.schema.json" }
+      "items": { "$ref": "IndicatorMetadata.schema.json" }
+    },
+    "barchart_metadata": {
```

Comment:
rob has clarified this is already included in the response, the schema allows additional properties and this was passed through but undescribed